okapi-core/src/test/java/org/folio/okapi/service/impl/PostgresHandleTest.java

```diff
@@ -127,4 +127,17 @@ void rejectTlsv1_2(Vertx vertx, VertxTestContext vtc) {
     configure("ssl = on", "ssl_min_protocol_version = TLSv1.2", "ssl_max_protocol_version = TLSv1.2");
     new PostgresHandle(vertx, config()).getConnection().onComplete(vtc.failing(fail -> vtc.completeNow()));
   }
+
+  @Test
+  void scram256(Vertx vertx, VertxTestContext vtc) {
+    configure("ssl = on", "password_encryption = scram-sha-256");
+    new PostgresHandle(vertx, config()).getConnection()
+        .compose(connection -> {
+          String sql = "ALTER USER " + POSTGRESQL_CONTAINER.getUsername()
+              + " WITH PASSWORD '" + POSTGRESQL_CONTAINER.getPassword() + "';";
+          return connection.query(sql).execute().mapEmpty();
+        })
+        .compose(x -> new PostgresHandle(vertx, config()).getConnection())
+        .onComplete(vtc.succeedingThenComplete());
+  }
```

Comment:
`POSTGRESQL_CONTAINER.getUsername()` is used by other test methods and should not be touched.
The `pg_authid` table reveals the actual method.
```
  @Test
  void scram256(Vertx vertx, VertxTestContext vtc) {
    configure("password_encryption = scram-sha-256");
    JsonObject config = config();
    config.remove("postgres_server_pem");
    new PostgresHandle(vertx, config()).getConnection()
        .compose(connection -> {
            return connection.query("CREATE USER scram PASSWORD 'foo'").execute()
                .compose(x -> connection.query("SELECT rolpassword FROM pg_authid WHERE rolname='scram'").execute())
                .map(rowSet -> rowSet.iterator().next().getString(0));
        })
        .onComplete(vtc.succeeding(password -> assertThat(password).startsWith("SCRAM-SHA-256")))
        .compose(x -> {
          config.put("postgres_username", "scram").put("postgres_password", "foo");
          return new PostgresHandle(vertx, config).getConnection();
        })
        .onComplete(vtc.succeedingThenComplete());
```

Reply:
ok. Disabling ssl did not work .. and, frankly, why should we check this.

Reply:
Disabling ssl works if replacing config() by config in line 138.